data/en/mbs_0205.json

```diff
@@ -504,4 +504,4 @@
             }]
         }
     ]
-}
+}
```

Comment:
Is there a reason you're removing all the newline characters from each of these files?
If not, re-add it to each of the files you've removed it from.

Reply:
@insacuri We don't use newline characters for our schemas. They get automatically removed if you run `yarn format`.

Reply:
They shouldn't be in this PR though.

Reply:
I think for minor things like this, does it really matter? It shouldn't have gone into master with those changes in the first place. But I ran `yarn format` to format my json schema but I feel it is counter-intuitive for me to add lines back in because it was missed out before.

I can add the lines back in, I don't mind but we need to either get rid of `yarn format` or make it a necessary check when linting.

Reply:
If we ran a linter over it and it was supposed to remove them, then it's fine being in this PR.  You're right, they shouldn't have gone into master like this anyway so we might as well sort it out while we're here